components/Auth/ForgotPassword.tsx

```diff
@@ -0,0 +1,105 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import { Button, HStack, Input, Text, useToast, VStack } from '@chakra-ui/react'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useHookFormProps } from 'hooks'
+import { AuthState } from 'models/users'
+import React, { Dispatch, SetStateAction } from 'react'
+import { useForm } from 'react-hook-form'
+import { resetPassword } from 'utils/firebase'
+import * as yup from 'yup'
+
+interface ForgotPasswordProps {
+  setCurrentState: Dispatch<SetStateAction<AuthState>>
+}
+
+interface ForgotPasswordForm {
+  email: string
+}
+
+const loginFormSchema = yup.object({
+  email: yup.string().required().email(),
+})
+
+const initialState = {
+  email: '',
+}
+
+const ForgotPassword = ({
+  setCurrentState,
+}: ForgotPasswordProps): JSX.Element => {
+  const toast = useToast()
+
+  const form = useForm({
+    defaultValues: initialState,
+    resolver: yupResolver(loginFormSchema),
+  })
+
+  const onSubmit = async ({ email }: ForgotPasswordForm) => {
+    const response = await resetPassword(email)
+
+    if (response.status === 200) {
+      toast({
+        title: 'Password Reset Email Sent',
+        description: response.message,
+        status: 'success',
+        duration: 5000,
+        isClosable: true,
+      })
+      setCurrentState('log_in')
+    } else {
+      toast({
+        title: 'Error',
+        description: response.message,
+        status: `error`,
+        duration: 5000,
+        isClosable: true,
+      })
+    }
+  }
+
+  return (
+    <VStack flex={1} w="100%" justifyContent="space-between" color="black">
+      <HStack>
+        <ArrowBackIcon
+          boxSize={6}
+          cursor="pointer"
+          onClick={() => setCurrentState('log_in')}
+          _hover={{ color: 'teal' }}
+          _focus={{ color: 'teal' }}
+        />
+        <Text fontSize={'3xl'} fontWeight="bold">
+          Reset Your Password
+        </Text>
+      </HStack>
+
+      <VStack w="100%" maxW="xs" as="form">
+        <VStack w="100%">
+          <Input
+            type="email"
+            isRequired
+            {...useHookFormProps('email', form)}
+            _invalid={{ borderColor: 'red', outlineColor: 'none' }}
```

Comment:
I think if you use [our `Input` component](https://github.com/MutualAidNYC/services-lists/blob/main/components/Forms/Input.tsx) that's wrapped in [Chakra's form control](https://v1.chakra-ui.com/docs/components/form/form-control#error-message), you don't need to manually add an error border.

Reply:
I meant to import from that but VSCode auto-imported from chakra will fix it!

Reply:
If you're using our `Input`, the border should highlight if [the `isInvalid` prop is true](https://v1.chakra-ui.com/docs/components/form/form-control#error-message) - no need to use `_invalid`.